### ISSUE TITLE
Error When Rebuilding When Building With Ahead of Time Compilation

### ISSUE THREAD

**Issue by gregfletch** [bugsnag/bugsnag-js#523]
When building an Angular application with the ahead of time compilation flag (`--aot`) set (i.e. `ng build --aot --watch` or `ng serve --aot`), the initial build succeeds, but subsequent builds fail with an error like:

```
ERROR in ./src/app/app.module.ngfactory.js
Module not found: Error: Can't resolve '@bugsnag/browser/dist/types/bugsnag' in '/Users/user/dev/ng-app/src/app'
```

Versions:

@bugsnag/core: 6.1.0
@bugsnag/js: 6.1.0
@bugsnag/plugin-angular: 6.1.0
Angular CLI: 7.3.8
Angular: 7.2.13

**Comment by PACMAN49** [comment]
i m facing the same issue with
@bugsnag/core: 6.2.0
@bugsnag/js: 6.2.0
@bugsnag/plugin-angular: 6.2.0
Angular CLI: 7.3.8
Angular: 7.0.2

**Comment by bengourley** [comment]
Thanks for the report. This seems like an issue with Angular's AOT mode. The second time it builds, for some reason, it looks for a `.ts|.tsx|.js|.mjs` file in `node_modules/@bugsnag/browser/dist/types` but there's only `.d.ts` files so is not satisfied. By adding an empty `.ts` file to this directory, the compiler gets tricked into thinking everything is ok and it all worked. This fix is added in #539.
